public/app/features/dashboard-scene/scene/PanelMenuBehavior.tsx

```diff
@@ -88,6 +94,35 @@ export function panelMenuBehavior(menu: VizPanelMenu) {
   asyncFunc();
 }
 
+function addDataTrailAction(dashboard: DashboardScene, vizPanel: VizPanel, items: PanelMenuItem[]) {
```

Comment:
Could we move this function and its dependencies to somewhere under `public/app/features/trails/...`.